Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:2.7
+
+RUN mkdir -p /opt/ecs-rollover
+COPY requirements.txt /opt/ecs-rollover
+RUN pip install -r /opt/ecs-rollover/requirements.txt
+
+COPY src/__init__.py /opt/ecs-rollover/
```

Comment:
Could we do `COPY src/*.py /opt/ecs-rollover/` here, or might that lead to unwanted behavior?